Formula/netdata.rb

```diff
@@ -0,0 +1,71 @@
+class Netdata < Formula
+  desc "Distributed real-time performance and health monitoring"
+  homepage "https://my-netdata.io/"
+  url "https://github.com/firehol/netdata/releases/download/v1.9.0/netdata-1.9.0.tar.bz2"
+  sha256 "542b4799aed1ee03b0a0dbd00fae988a788622c431382e6429362fbbb4f0f017"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ossp-uuid"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+
+    (etc/"netdata").install "system/netdata.conf"
+  end
+
+  def post_install
+    config = etc/"netdata/netdata.conf"
+    return unless File.exist?(config)
```

Comment:
Why would the file not exist? We've just installed it above

Reply:
Dropped